packages/js/src/Modules/Verto/index.ts

```diff
@@ -64,6 +65,7 @@ export default class Verto extends BrowserSession {
       this.sessionid = response.sessid;
       sessionStorage.setItem(SESSION_ID, this.sessionid);
       trigger(SwEvent.Ready, this, this.uuid);
+      logger.info('Session Ready!');
```

Comment:
What is the purpose of this log?

Reply:
I copied from the method `_onSocketOpen` that `Blade` was using, I think it is helpful when you are debugging the service on production.

Reply:
You can check here https://github.com/team-telnyx/webrtc/pull/75/files#diff-68dd50822bd145df05bdf53bbc4b6e2242cafe1e4d03aa94f2a886babcc1400dL287